utils/scraper_processing_utils.R

```diff
@@ -90,7 +90,7 @@ format_author_firstnames <- function(author_vec){
     author_vec = gsub("[.]", "", author_vec)
 
     # or less than 3 characters
```

Comment:
```suggestion
    # or less than 2 characters
```
To keep the documentation consistent